Source/Logging.swift

```diff
@@ -28,69 +26,65 @@ public class RxBluetoothKitLog: ReactiveCompatible {
         case error = 4
     }
 
-    /// Set new log level.
-    /// - Parameter logLevel: New log level to be applied.
-    public static func setLogLevel(_ logLevel: LogLevel) {
```

Comment:
`setLogLevel` and `getLogLevel` methods should be left in `RxBluetoothKitLog` class. These methods are used in some projects and removing them will cause these projects will not compile. My suggestion is to add `setLogLevel` and `getLogLevel` methods to `Logger` protocol what will enable to use them in this class.